### PR TITLE
add total_cpu_time_saved to invocation trend response

### PR DESCRIPTION
**Pull request by luluz66**
**Related issues**: N/A

### CODE REVIEW

File: enterprise/server/invocation_stat_service/invocation_stat_service.go [hunk 1]
```diff
@@ -112,7 +112,8 @@ func (i *InvocationStatService) getTrendBasicQuery(timezoneOffsetMinutes int32)
 	    SUM(total_download_size_bytes) as total_download_size_bytes,
 	    SUM(total_upload_size_bytes) as total_upload_size_bytes,
 	    SUM(total_download_usec) as total_download_usec,
-        SUM(total_upload_usec) as total_upload_usec
+        SUM(total_upload_usec) as total_upload_usec,
+		SUM(total_cached_action_exec_usec) as total_cpu_micros_saved
```
Comment (jdhollen) [inline enterprise/server/invocation_stat_service/invocation_stat_service.go:116]: ahaha i don't know what's going on with the indentation here, but can you make this match the previous lines?

Comment (luluz66) [reply enterprise/server/invocation_stat_service/invocation_stat_service.go]: uhhh tabs..Fixed

Comment (jdhollen) [reply enterprise/server/invocation_stat_service/invocation_stat_service.go]: ahahaha i don't want to keep nitpicking this but these last 3 lines are still misaligned relative to the lines above it and i assume your editor is just aligning to whatever came on the line before--
image

**Review by jdhollen — APPROVED** [review]
just my little nitpick, so i'm approving.

**Pull request merged**